x/stake/keeper/validator.go

```diff
@@ -403,18 +400,20 @@ func (k Keeper) getPowerIncreasing(ctx sdk.Context, oldFound bool, oldValidator,
 
 // get the bond height and incremented intra-tx counter
 // nolint: unparam
-func (k Keeper) bondIncrement(ctx sdk.Context, oldFound bool, oldValidator,
-	newValidator types.Validator) (height int64, intraTxCounter int16) {
+func (k Keeper) bondIncrement(
+	ctx sdk.Context, found bool, validator types.Validator) (height int64, intraTxCounter int16) {
```

Comment:
Can we still call this `oldValidator`? It was confusing to read through the first time because I thought you changed the logic

Reply:
For sure!